tests/src/main/resources/tck-audit.xml

```diff
@@ -704,10 +707,7 @@
         <assertion id="f">
             <text>Container element constraints can be applied to nested container types</text>
         </assertion>
-        <assertion id="g">
```

Comment:
On that removal you said it's covered by some other assertion now. On that end, did you see my recent example addition of `Map<String, Map<@Valid AddressType, @Valid Address>>`? That'd be a good test case to have, too (two inner extractors).